android/KMEA/app/src/main/java/com/tavultesoft/kmea/util/FileUtils.java

```diff
@@ -383,11 +385,21 @@ public static boolean hasJavaScriptExtension(String filename) {
     return f.endsWith(JAVASCRIPT);
   }
 
+  public static boolean hasLexicalModelExtension(String filename) {
+    String f = filename.toLowerCase();
+    return f.endsWith(LEXICALMODEL);
+  }
+
   public static boolean hasKeyboardPackageExtension(String filename) {
     String f = filename.toLowerCase();
     return f.endsWith(KEYBOARDPACKAGE);
   }
 
+  public static boolean hasLexicalModelPackageExtension(String filename) {
```

Comment:
I want to treat `.model.kmp` as convention in the lexical-models repository and not as a requirement. This means that Keyman for Android must test the content of kmp.json to determine whether a package is a lexical model package or a keyboard package, and at that point it can determine where to put the file. You may decide to put all packages in the same filesystem location instead and use separate maps to index them?